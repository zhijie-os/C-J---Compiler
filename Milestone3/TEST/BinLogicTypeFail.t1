main()
{
	boolean x;
	int y;
	boolean z;
	z= x && y;
}
