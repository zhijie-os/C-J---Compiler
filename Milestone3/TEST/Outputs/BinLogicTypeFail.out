Line 6, binary boolean operator && has Non BOOLEAN type on RHS..
